### PR TITLE
PageHero & ActionCard image refactors

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Refactor for the `PageHero` & `ActionCard` components to display the images in a better way.

- `PageHero`: added a more granular `sizes` prop.
- `ActionCard`: added a new prop `imageWidth` to control the desired width from the different pages.

### CODE REVIEW

File: src/components/Hero/HomeHero/index.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ const HomeHero = ({ heroImg }: HomeHeroProps) => {
         <Image
           src={heroImg}
           alt={t("page-index:page-index-hero-image-alt")}
+          sizes="100vw"
```
Comment (wackerow) [inline src/components/Hero/HomeHero/index.tsx:20]: The Next.js docs suggest adding the `style` prop with `{width: "100vw", height: "auto"}` for responsive images... Will this be handled appropriately with just `w="full" h="full"`?

File: src/components/ActionCard.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { StaticImageData } from "next/image"
```
Comment (wackerow) [inline src/components/ActionCard.tsx:1]: Mentioned in Discord, but is `StaticImageData` our preferred approach to typing the image `src`?

**Review by wackerow — APPROVED** [review]
Left a couple questions, but functionally seems to be working great! Thanks @pettinarip!

File: src/components/ActionCard.tsx [hunk 3]
```diff
@@ -76,7 +79,7 @@ const ActionCard = ({
       >
         <Image
           src={image}
-          width={220}
+          width={imageWidth}
```
Comment (wackerow) [inline src/components/ActionCard.tsx:82]: Seeing an image height bug now:

image


Canonical:
image

```suggestion
          width={imageWidth}
          maxH="full"
```

@pettinarip  This does the trick, but does that seem like an appropriate approach to you?

**Review by wackerow — CHANGES_REQUESTED** [review]

File: src/components/ActionCard.tsx (hunk 3 above)
Comment (pettinarip) [reply src/components/ActionCard.tsx:82]: Nice catch! that solution looks perfect

File: src/components/Hero/HomeHero/index.tsx (hunk 1 above)
Comment (pettinarip) [reply src/components/Hero/HomeHero/index.tsx:20]: Well, I've tested it and worked well. Those are the style props we are setting in prod. Not sure if the Next suggestion would have a different outcome tbh. At the end, they are css rules, and I'd prefer to use as much as we can the Chakra style props if possible. LMK if you have any concerns about using this approach.

**Review by wackerow — APPROVED** [review]
lgtm!

**Review by corwintines — APPROVED** [review]

**Pull request merged**